homeassistant/components/qwikswitch.py

```diff
@@ -33,7 +33,10 @@
         vol.Required(CONF_URL, default='http://127.0.0.1:2020'):
             vol.Coerce(str),
         vol.Optional(CONF_DIMMER_ADJUST, default=1): CV_DIM_VALUE,
-        vol.Optional(CONF_BUTTON_EVENTS): vol.Coerce(str)
+        vol.Optional(CONF_BUTTON_EVENTS): cv.ensure_list_csv,
+        vol.Optional(CONF_SENSORS, default=[]): vol.Schema({cv.slug: str}),
```

Comment:
The default value must be able to pass the validation schema.

Reply:
Missed this completely!